### PR TITLE
Better user experience.

### PR DESCRIPTION
**Pull request by rahularya50**

### CODE REVIEW

File: static/js/interpreter.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
 import {execute} from "./execution.js";
 
 const introText =
-`Welcome to the 61A SQL interpreter!
-Enter some SQL below:
+`61A SQL Web Interpreter
+<a href="https://github.com/Cal-CS-61A-Staff/queryable">on GitHub</a>
+--------------------------------------------------------------------------------
+Welcome to the 61A SQL interpreter!
+Type <code>.help</code> for instructions, or <code>.read</code> to load a file.
```
Comment (kavigupta) [inline static/js/interpreter.js]: load a file from where?

**Review by kavigupta — CHANGES_REQUESTED** [review]

File: static/js/interpreter.js (hunk 1 above)
Comment (rahularya50) [reply static/js/interpreter.js]: It opens a file upload dialog - at least, it does for me on MacOS using Chrome.

Comment (kavigupta) [reply static/js/interpreter.js]: very unintuitive error messages

image

Comment (kavigupta) [reply static/js/interpreter.js]: I think say run "run `.read` to select a file from your computer"

**Review by kavigupta — APPROVED** [review]

**Review by kavigupta — APPROVED** [review]

**Pull request merged**